requirements-optional.txt

```diff
@@ -12,3 +12,4 @@ tesserocr>=1.2; platform_system != 'Windows'
 akismet>=1.0
 boto3
 zeep
+social-auth-core>=3.1.0
```

Comment:
Not needed at all here, it's alredy in `requirements.txt`.

Reply:
My bad. Removed.